src/components/transaction/AddPartiesWrapper/index.jsx

```diff
@@ -105,8 +119,79 @@ export default function AddPartiesWrapper() {
         ? [...userIdsAndRoles, ...modalData?.invitedUsers]
         : userIdsAndRoles,
     });
+
+    // add transaction assignees
+    await addTransactionAssignees({
+      transactionId: transactionData.id,
+      userIdsAndRoles: modalData?.invitedUsers?.length
+        ? [...userIdsAndRoles, ...modalData?.invitedUsers]
+        : userIdsAndRoles,
+    });
+
+    // add the note
+    await addTransactionNote({
+      notes,
+      transactionId: transactionData.id,
+      workflowStepId: workflowStep.id,
+    });
+
+    // navigate to step 2
+    navigate(`/transaction/${listingId}/${transactionStepsNames.assignTasks}`);
+  };
+
+  /**
+   * fetch Transaction Data function
+   */
+  const fetchTransactionData = async () => {
+    // reset modal data
+    setModalData(null);
+
+    // get step info
+    const step = await getWorkflowStep(1);
```

Comment:
```suggestion
    const step = await getWorkflowStep(steps.firstStep);
```

can this function be changed to take something like this